MaxText/configs/cpu/16b.sh

```diff
@@ -0,0 +1,28 @@
+echo "Running 16b.sh"
```

Comment:
Please put these in end to end and run some as CPU only end to end tests.

I also don't htink ou need the weird argument copy paaa (or even the dupliction at all nowadays with the export M_whatever.